### DISCUSSION TITLE
第5节课笔记(13班)

### DISCUSSION THREAD

**Discussion by Lum1104** [InternLM/Tutorial#371] [笔记]
13班第五节课笔记

**Comment by qingyin741** [comment]
https://zhuanlan.zhihu.com/p/678108227

**Comment by SiliconSisyphus** [comment]
https://www.notion.so/60f62dcdd0a548beacc03a02969d5ac2

**Comment by BOOMO4** [comment]
https://l055jh6h54r.feishu.cn/wiki/He1awmxLRiw3jskdwJscLPEqnGe?from=from_copylink

**Comment by Xrcb** [comment]
https://blog.csdn.net/weixin_64546620/article/details/135663659
